website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,183 @@ curl \
+| ------- | ----------------------------------------------- | ------------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "data-retention-policy-delete-olders" or "data-retention-policy-dont-deletes"`) | The request was successful. [Types of Data Retention Policies](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies). |
+| [404][] | [JSON API error object][]                       | Organization not found, data retention policy does not exist, or user unauthorized to perform action |
+
+### Request Body
+
+No request body.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 60
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        }
+      }
+    }
+  }
+}
+```
+
+## Create or update data retention policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`POST /organizations/:organization_name/relationships/data-retention-policy`
+
+This endpoint creates a data retention policy for an organization, or updates the existing policy. Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
+
+The endpoint can also be used to change the type of the data retention policy on the organization. Making a request with the new type of policy will automatically create a data retention policy of the new type and replace the existing organization policy. For more information on the types of data retention policies, refer to [Data Retention Policy Types](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies).
+
+| Parameter            | Description                                                                |
+| -------------------- | -------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to update the data retention policy for.      |
+
+| Status  | Response                  | Reason(s)                                                      |
+| ------- | ------------------------- | -------------------------------------------------------------- |
+| [204][] | No Content                | Successfully updated the organization data retention policy.   |
+| [404][] | [JSON API error object][] | Organization not found, or user unauthorized to perform action.|
```

Comment:
Non-Blocking Nitpick: Should we add a space here between the period and the pipe?:
```suggestion
| [404][] | [JSON API error object][] | Organization not found, or user unauthorized to perform action. |
```